app/services/flow/flows/vehicle_steps.rb

```diff
@@ -0,0 +1,28 @@
+module Flow
+  module Flows
+    class VehicleSteps < FlowSteps
```

Comment:
The vehicle steps pushed `ProviderCapital` flow class over the size limit recommended by Rubocop - so split the vehicle steps into their own flow.